pkg/apis/api/types.go

```diff
@@ -1102,9 +1116,22 @@ type ComponentReportTriageIncidentSummary struct {
 	TriagedIncidents []TriagedIncident `json:"incidents"`
 }
 
+// TestRegression is used for rows in the test_regressions table and is used to track when we detect test
+// regressions opening and closing.
+type TestRegression struct {
+	Release      string                 `bigquery:"release" json:"release"`
+	TestID       string                 `bigquery:"test_id" json:"test_id"`
+	TestName     bigquery.NullString    `bigquery:"test_name" json:"test_name"`
```

Comment:
Can the test name really be null?

Reply:
Only because I added it after making the table. :) I'll go recreate it.